app/profile/page.js

```diff
@@ -75,27 +75,28 @@ export default function EditProfile() {
             const userProfileRef = doc(db, "users", uid);
 
             const userProfile = {
-                firstName,
-                lastName,
+                first_name: firstName,
+                last_name: lastName,
                 email, 
                 birthday,
                 country,
                 village,
                 bio,
-                educationLevel,
-                isOrphan: isOrphan === 'Yes' ? true : false,
-                livesWith,
-                dreamJob,
+                education_level: educationLevel,
+                is_orphan: isOrphan === 'Yes' ? true : false,
```

Comment:
is this a text input or a dropdown/radio button? If it's not a dropdown we should make this lowercase to avoid this failing inaccurately

Reply:
Oh, so you mean the following?
```suggestion
                is_orphan: isOrphan.toLowerCase() === 'yes' ? true : false,
```

Reply:
yeah correct!